x-pack/test/security_solution_cypress/cypress/e2e/investigations/timelines/esql/esql_assistant.cy.ts

```diff
@@ -21,156 +21,160 @@ const KQL_QUERY = '_index : "auditbeat-*" and ecs.version : "8.0.0"';
 const EQL_QUERY = 'process where process.name == "zsh"';
 
 // Failing: See https://github.com/elastic/kibana/issues/180756
-describe.skip('Basic Assistant tests', {
-  tags: ['@ess'],
-  env: {
-    ftrConfig: {
-      kbnServerArgs: [
-        `--xpack.actions.preconfigured=${JSON.stringify({
-          'preconfigured-openai': {
-            name: 'preconfigured-openai',
-            actionTypeId: '.gen-ai',
-            config: {
-              apiProvider: 'OpenAI',
-              apiUrl: 'https://api.openai.com/v1/chat/completions',
-              defaultModel: 'gpt-4',
+describe.skip(
```

Comment:
Are these changes supposed to be in this PR?

Reply:
these changes were automatically applied as part of a307c7756539882a2c3eb0d84f622660fab6fe86 commit from CI - I will find out why and should they be included.